README.md

```diff
@@ -14,6 +14,7 @@ For more information, see [the Codewind website](https://www.eclipse.org/codewin
 - Pull in your existing applications and use Codewind to help get them cloud ready.
 - See code changes reflected in your containerized application almost instantaneously. The Codewind advanced inner loop manages your application updates efficiently.
 - Develop in containers without feeling like you're developing in containers. Development flow within the IDE feels the same as traditional application development.
+- Without leaving the comfort of your local IDE, iteratively develop on Kubernetes and interact with Kubernetes services.
```

Comment:
I'm not sure if "services" is the right vocabulary. Services are a type of resource in Kube (like how a pod or an ingress is a resource) and Codewind doesn't have any features that pertain to services, other than using them internally for implementation.

Reply:
`Kubernetes resources`?